chatwoot.yaml

```diff
@@ -11,51 +11,133 @@ common:
     source: https://github.com/chatwoot/chatwoot
     publisher: "monk.io"
     icon: https://storage.googleapis.com/template_logos/chatwoot-image.png
-
+    email: maciej@monk.io
   containers:
     defines: containers
     rails:
       image: chatwoot/chatwoot:latest
       environment:
         - NODE_ENV=production
-        - RAILS_ENV=production
         - DEFAULT_LOCALE=en
-        - FRONTEND_URL=http://0.0.0.0:3000
+        - <- `REDIS_URL=redis://${redis-host}:6379`
         - <- `SECRET_KEY_BASE=${secret-key-base}`
         - <- `REDIS_URL=redis://${redis-host}:6379`
         - <- `POSTGRES_HOST=${db-host}`
-        - <- `POSTGRES_USERNAME=${db-user}`
-        - <- `POSTGRES_PASSWORD=${db-password}`
-        - <- `SMTP_ADDRESS=${smtp-host}`
+        - <- `POSTGRES_USERNAME=${postgres-db-user}`
+        - <- `POSTGRES_PASSWORD=${postgres-db-password}`
+        - <- `MAILER_SENDER_EMAIL=${mailer-sender-email}`
+        - <- `SMTP_ADDRESS=${smtp-address}`
         - <- `SMTP_PORT=${smtp-port}`
-        - <- `SMTP_USERNAME=${smtp-user}`
+        - <- `SMTP_USERNAME=${smtp-username}`
         - <- `SMTP_PASSWORD=${smtp-password}`
+        - <- `SMTP_AUTHENTICATION=${smtp-authentication}`
+        - <- `SMTP_ENABLE_STARTTLS_AUTO=${smtp-enable-startttls-auto}`
+        - <- `SMTP_DOMAIN=${smtp-domain}`
+        - <- `REDIS_PASSWORD=${redis-password}`
+        - <- `FRONTEND_URL=${server-name}`
```

Comment:
The expectation of FRONTEND_URL is to have the full URL include `https://` .  I think  server-name returns just the domain name right ? .  I am not sure whether this will break anything. 

should we update ?

Reply:
good catch, I will update it. I will setup test domain in cloudflare today and will test it against it.